src/app/core/feature.js

```diff
@@ -35,10 +35,9 @@ export default class Feature {
      * Returns the reference object id
      * @return {String}
      * @memberof Feature
-     * @private
      */
     get referenceID() {
-        return this.__referenceObject;
+        return this.__referenceID;
```

Comment:
I believe '__referenceObject' us just an incorrect reference to 'referenceID'

Reply:
yup ! I guess this fell through the cracks